lib/ansible/modules/system/iptables.py

```diff
@@ -399,8 +399,14 @@ def construct_rule(params):
         False)
     append_match(rule, params['comment'], 'comment')
     append_param(rule, params['comment'], '--comment', False)
-    append_match(rule, params['ctstate'], 'state')
-    append_csv(rule, params['ctstate'], '--state')
+    if params['ctstate']:
```

Comment:
I am using this yaml file:
>- hosts: localhost
  tasks:
     - iptables:
         chain: INPUT
         match: state
         state: ESTABLISHED,RELATED
         jump: ACCEPT